wosfile/tags.py

```diff
@@ -77,6 +77,7 @@
     ("UT", "Unique Article Identifier", False, False),
     ("VL", "Volume", False, False),
     ("WC", "Web of Science Categories", True, False),
+    ("WE", "???", False, False),
```

Comment:
While I cannot find any documentation for this new field, it seems like it contains the index(es) the record is taken from.
```suggestion
    ("WE", "Web of Science indexes", True, False),
```